### ISSUE TITLE
Proxy Support

### ISSUE THREAD

**Issue by gappleto97** [maks/MGit#160]
It would be nice if you could tunnel this app through I2P or TOR. This is easily doable if there is proxy support.

**Comment by maks** [comment]
I'm not sure what is involved to support this - is it just support for an an Http proxy?

**Comment by gappleto97** [comment]
Yes. That should be enough for TOR. I think it would also work for I2P, but I'm less knowledgeable on that front.
